homeassistant/components/nobo_hub/sensor.py

```diff
@@ -46,6 +46,7 @@ class NoboTemperatureSensor(SensorEntity):
     _attr_native_unit_of_measurement = UnitOfTemperature.CELSIUS
     _attr_state_class = SensorStateClass.MEASUREMENT
     _attr_should_poll = False
+    _attr_has_entity_name = True
```

Comment:
Don't you have to set `_attr_name = None` as well?

Reply:
Nope, since 2023.7.0 you only have to do that for entities that share the exact same name as the device, eg hubs, covers, lights. 

This is not needed here since (also since 2023.7.0), the entity will use the device class to get it's (translated) name. Since the device class here is Temperature, the result would be `{device} Temperature`

Reply:
Ah, thanks! I hadn't fully grasped this change.

Reply:
And we also now use device classes for this with default translations, because otherwise a lot of integrations would add the same string over and over again :)